src/Material/Textfield.elm

```diff
@@ -99,7 +99,7 @@ type alias Config m =
   , cols : Maybe Int
   , autofocus : Bool
   , maxlength : Maybe Int
-  , style : List (Options.Style m)
+  , inner : List (Options.Style m)
```

Comment:
Textfield actually does not expose `Config` so this is a `MINOR` change according to `elm-package diff`